enterprise/models.py

```diff
@@ -2511,12 +2513,41 @@ class DefaultEnterpriseEnrollmentIntention(TimeStampedModel, SoftDeletableModel)
     )
     history = HistoricalRecords()
 
+    def get_cached_content_metadata(self):
```

Comment:
[inform] Based on our discussion, the underlying `EnterpriseCatalogAPIClient` method to be called (i.e., `get_content_metadata_content_identifier`) has a try/except with reasonable returns of empty dict. I think we can nix this helper method and directly call `get_and_cache_catalog_content_metadata` where needed.